### PR TITLE
dotnetconf2022 Bengaluru changes

### PR DESCRIPTION
**Pull request by PranamBhat**
dotnetconf2022 Bengaluru changes

### CODE REVIEW

**Comment by PranamBhat** [comment]
@svswaminathan dotnetconf2022 Bengaluru changes. There are few minor changes are remaining.

File: dotnet-conf-2022/index.html [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+	<meta name="twitter:image" content="http://bdotnet.in/dotnet-conf-2022/assets/images/dotnet-conf-2022-banner.PNG">
+
+
+	<!-- Open Graph data -->
+	<meta property="og:title" content=".NET Conf 2022 - Bengaluru" />
+	<meta property="og:type" content="website" />
+	<meta property="og:url" content="https://bdotnet.in/dotnet-conf-2022/" />
+	<meta property="og:image" content="http://bdotnet.in/dotnet-conf-2022/assets/images/dotnet-conf-2022-banner.PNG" />
+	<meta property="og:description"
+		content=".NET Conf 2022 is a free, virtual developer conference co-organized by the Bangalore DotNet UG (BDotNet), Dot Net Learners House community and Microsoft | Bengaluru, IN | JAN 23 2022" />
```
Comment (swaminathanvetri) [inline dotnet-conf-2022/index.html]: Date of the event is Jan 7,2023

File: dotnet-conf-2022/index.html [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+										<a href="https://www.microsoft.com" target="_blank">
+											<img src="assets/images/Microsoft.png" />
+										</a>
+									</div>
+									<div class="col-md-3">
+										<a href="https://dotnetfoundation.org/" target="_blank">
+											<img src="assets/images/dotnetfoundationhorizontal.svg" />
+										</a>
+									</div>
+									<div class="col-md-3">
```
Comment (swaminathanvetri) [inline dotnet-conf-2022/index.html:479]: Tech platform can be removed from sponsors

File: dotnet-conf-2022/index.html [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+											<a href="https://www.youtube.com/user/BdotnetTV" target="_blank"><i
+													class="fa fa-youtube"></i></a>
+											<a href="https://twitter.com/bdotnetug" target="_blank"><i
+													class="fa fa-twitter"></i></a>
+										</div>
+									</div>
+								</div>
+								<div class="col-md-6" style="text-align:center;margin-top:4%;">
+									<div class="row">
+										<h1>Dot Net Learners House</h1>
```
Comment (swaminathanvetri) [inline dotnet-conf-2022/index.html]: Dotnet learner's house related details can be removed as well

**Review by swaminathanvetri — CHANGES_REQUESTED** [review]
Added a couple of minor comments. Pls address then

**Comment by swaminathanvetri** [comment]
I also see quite a few test.txt files. Can you pls clean them up as well ??

File: dotnet-conf-2022/index.html (hunk 1 above)
Comment (PranamBhat) [reply dotnet-conf-2022/index.html]: Okay. Done.

File: dotnet-conf-2022/index.html (hunk 2 above)
Comment (PranamBhat) [reply dotnet-conf-2022/index.html:479]: Tech platform removed from sponsors

File: dotnet-conf-2022/index.html (hunk 3 above)
Comment (PranamBhat) [reply dotnet-conf-2022/index.html]: Dotnet learner's house related details removed

**Comment by PranamBhat** [comment]
Addressed all the comments and resolve the same.

**Comment by PranamBhat** [comment]
Addressed all the comments and resolved the same.

File: dotnet-conf-2022/index.html [hunk 4]
```diff
@@ -6,25 +6,25 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-	<title>.NET Conf 2022 - Bengaluru</title>
-	<meta name=".NET Conf 2022" content="width=device-width, initial-scale=1">
+	<title>.NET Conf 2023 - Bengaluru</title>
```
Comment (swaminathanvetri) [inline dotnet-conf-2022/index.html:9]: Event Name is .NET Conf 2022 only.. the date should be changed to Jan 7, 2023

**Review by swaminathanvetri — CHANGES_REQUESTED** [review]

**Pull request merged**